source/common/upstream/cluster_manager_impl.cc

```diff
@@ -109,7 +110,10 @@ void ClusterManagerInitHelper::maybeFinishInitialize() {
       for (auto iter = secondary_init_clusters_.begin(); iter != secondary_init_clusters_.end();) {
         Cluster* cluster = *iter;
         ++iter;
-        cluster->initialize();
+        cluster->initialize([cluster, this] {
```

Comment:
cluster.initialize(initialize_cb);? or am I missing something suble

Reply:
initialize_cb doesn't exist in this scope. I tried making it a member variable but cluster needs to be captured so I decided to just copy it here. I don't think it's worth trying to do something better.